src/user/main.cpp

```diff
@@ -191,7 +211,12 @@ void setup() {
             #endif
 
             // We need a little delay to allow the motor time to process if it needs it
+            #ifdef ENABLE_BLINK
+            // ! Only for testing
+            blink();
+            #else
             delay(50);
+            #endif
         }
     }
```

Comment:
Why a little delay? The motor updates are done in interrupts, so the motor shouldn't have any problems.

Reply:
I agree with you. But delay(50) presents in the master branch.

Reply:
Where is the delay(50)?

Reply:
https://github.com/CAP1Sup/Intellistep/blob/bffbdc91923994ea7d725eebc85585b0159439e9/src/user/main.cpp#L194-L195